src/Components/NavBar/NavBar.tsx

```diff
@@ -232,6 +231,8 @@ export const NavBar: React.FC = track(
   )
 })
 
+export const NavBarHeight = 60 - 1 // border offset
```

Comment:
1px nudges like this (and the -1px margin nudge below) make me wonder if something is wrong somewhere else.

Reply:
Also - does `space(6) - 1` work, instead of `60 - 1`? That would feel more respectful* of the design system.

(Respectful is the wrong word, but I can't figure out the work I'm looking for. Basically, having a hard-coded 60 feels like we are abandoning the design system, instead of using it to help us define spacing. It probably doesn't matter much, but it feels like all spacing should be driven by the design-system, in case we ever change it.)

Reply:
Ahh yes, I agree; will update 👍

Reply:
Re the `- 1` stuff, it's ugly and annoying and feels hacky :( The only reason it's needed is because there's an outline on the nav bar, and when we hover over nav items we want the menu to appear perfectly flush, aligning _its_ borders with the menu border so things don't double up.